src/app/shared/services/api/record.repo.ts

```diff
@@ -159,10 +159,28 @@ export class RecordRepo extends BaseRepo {
 
   public async update(recordVOs: RecordVO[], archiveId: number) {
     return await firstValueFrom(
-      this.httpV2.post('/record/update', { ...recordVOs[0], archiveId }, null),
+      this.httpV2.post(
+        '/record/update',
+        this.prepareUpdateRecordRequest(recordVOs, archiveId),
+        null,
+      ),
     );
   }
 
+  private prepareUpdateRecordRequest(recordVOs: RecordVO[], archiveId: number) {
```

Comment:
What type does this end up returning? (approving regardless)

Reply:
This is returning a thing that is like a `RecordVO` but with the cases swapped for what the endpoint is expecting. This probably could be a type called `UpdateRecordV2Request` or something like that, but since it's only used internally here I have not externally defined it.